### PR TITLE
91 p2 gmv parameter

### PR DESCRIPTION
**Pull request by kekuso**
- add implementation and tests for P2 GMV PARAMETER
- add fixes and tests for F178 ORWU CLINLOC, empty string param
- add Parameter_Entity-8989_518 object to vdm model file

### CODE REVIEW

File: prototypes/parameters/p02_gmvParameter.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+                    // if ent parameter is not valid, throw an Error
+                    if (!ent) throw new Error('Undefined local variable: ENT');
+                    if (ent === 'USR') resArray = [VDM.user().name.value, ''];
+                    else if (ent === 'SYS') {
+                        try {
+                            resArray = [VDM.describe('8989_3-1').domain_name.label, ''];
+                        } catch (e) {
+                            resArray = ['', ''];
+                        }
+                    } else if (ent === 'DIV') resArray = [VDM.user().division.value[0].division.label.split('/')[1], ''];
```
Comment (mfuroyama) [inline prototypes/parameters/p02_gmvParameter.js]: This:
```Javascript
VDM.user().division.value[0].division.label.split('/')[1]
```
looks ever-so-slightly dangerous.  Consider adding some error checks to prevent unhandled exceptions.
